### ISSUE TITLE
Text Classification Validation Accuracy does not change

### ISSUE THREAD

**Issue by rakshanda22** [keras-team/autokeras#946]
In the following text classification example [https://autokeras.com/examples/imdb/](url) the validation accuracy remains the same across all epochs and trials. The validation accuracy is 50.62% and seems like random classification accuracy. What could be the possible issue?

**Comment by haifeng-jin** [comment]
@rakshanda22

We will look into it. Thank you for the report.

**Comment by johny-b** [comment]
I noticed the same.

Also:
* all is well on autokeras 1.0.0, bug is only on 1.0.1
* works just as bad on other text datasets

I'm using Ubuntu 18.04 and tensorflow 2.1.0

**Comment by haifeng-jin** [comment]
@johny-b If so, it must be the text vectorization layer.
We didn't use it in 1.0.0 and use it in 1.0.1.

It's weights cannot be saved for now.
When the tuner loads it for eval and predict, it doesn't work.

https://github.com/tensorflow/tensorflow/blob/v2.1.0/tensorflow/python/keras/layers/preprocessing/text_vectorization.py#L290

Hopefully, it can be saved with TF 2.2.
I will confirm with the tensorflow team. If not, we may try to do a walk around.

**Comment by rakshanda22** [comment]
So is there a way to drop down to autokeras 1.0.0?

**Comment by haifeng-jin** [comment]
@rakshanda22 Yes.
```
pip3 uninstall autokeras
pip3 install autokeras==1.0.0
```

**Comment by SoYoungCho** [comment]
Having the same problem. Any suggestions?

**Comment by haifeng-jin** [comment]
@SoYoungCho This is a rather old issue. I believe it has been fixed already. Are you using AutoKeras 1.0.4 with Keras Tuner 1.0.2rc1? Thanks.

**Comment by SoYoungCho** [comment]
> @SoYoungCho This is a rather old issue. I believe it has been fixed already. Are you using AutoKeras 1.0.4 with Keras Tuner 1.0.2rc1? Thanks.

Problem solved. I did use AutoKeras 1.0.4 with Keras Tuner 1.0.2rc1, but the problem was overfitting. Randomly shuffling the dataset solved my issue. Thanks!
